src/domain/cellar/index.ts

```
@@ -0,0 +1,427 @@
+"server only";
+
+import { nanoid } from "nanoid";
+import { cache } from "react";
+
+import type { Prisma, ServingFrom } from "@db/client";
+
+import {
+  InvalidQuantityError,
```

ESLint

[error] 'InvalidQuantityError' is defined but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] 'InvalidQuantityError' is defined but never used.

(unused-imports/no-unused-imports)